### PR TITLE
Initial setup of repository

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
Here, I am following the setup instructions for a workshop-specific repository _or_ have filed issues where appropriate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,7 @@
-# training-specific-template
+# CCDL Virtual Workshop: 2020 June Training
```
Comment (jashapiro) [inline README.md]:
```suggestion
---
title: CCDL Virtual Workshop: 2020 June Training
---
```

File: README.md [hunk 2]
```diff
@@ -1,82 +1,7 @@
...
 
-This repository is intended to serve as a template for creating a repository for an individual CCDL workshop.
-The repository structure and use of GitHub pages is intended to gather all material required to administer a workshop in one user-friendly place. 
-We use a template repository approach for maintainability.
+**If you are a participant in in the training workshop and you've landed here, visit https://alexslemonade.github.io/2020-june-training/workshop/HOME to get started!**
```
Comment (jashapiro) [inline README.md:5]: I think I can make the header point here, as I think it will send us to the readme by default.

File: _config.yaml [hunk 3]
```diff
@@ -31,9 +31,12 @@ theme: minima
 header_pages:
   # Note that when first cloned, the `workshop` directory will not exist,
   # as described at https://github.com/AlexsLemonade/training-specific-template#customizing-the-new-repository-for-an-individual-training-workshop
+  - workshop/HOME.md
```
Comment (jashapiro) [inline _config.yaml:34]: I think I can make this better (have the main head point here). Will be in a separate PR.

File: _config.yaml [hunk 4]
```diff
@@ -31,9 +31,12 @@ theme: minima
...
   - workshop/SCHEDULE.md
   - workshop/workshop-structure.md
+  - workshop/software-setup.md
```
Comment (jashapiro) [inline _config.yaml:37]: These should be accompanied by changes to the frontmatter for these files to add a `nav_title` variable.

File: _config.yaml [hunk 5]
```diff
@@ -31,9 +31,12 @@ theme: minima
...
   - workshop/workshop-materials.md
+  - workshop/resources-for-consultation-sessions.md
 author: Childhood Cancer Data Lab
 # email: user@site
```
Comment (jashapiro) [inline _config.yaml]: Do we want our email in the footer?

**Review by jashapiro — COMMENTED** [review]
A few small changes, but looks good overall. I will modify the main header link next.

File: README.md (hunk 2 above)
Comment (jaclyn-taroni) [reply README.md:5]: Yes https://alexslemonade.github.io/2020-june-training will send us to the README.md by default, provided I am understanding you correctly.

File: _config.yaml (hunk 4 above)
Comment (jaclyn-taroni) [reply _config.yaml:37]: So for example we could use
```
---
title: Resources for Consultation Sessions
nav_title: Consultation Sessions
---
```
?

File: _config.yaml (hunk 5 above)
Comment (jaclyn-taroni) [reply _config.yaml]: Can add `training@ccdatalab.org`

File: _config.yaml (hunk 4 above)
Comment (jashapiro) [reply _config.yaml:37]: Correct.

File: README.md (hunk 1 above)
Comment (jaclyn-taroni) [reply README.md]: This appears to break things for me locally - I'm pointed to the index when I navigate to the server address (but perhaps this is intended).

File: _config.yaml (hunk 3 above)
Comment (jashapiro) [reply _config.yaml:34]: See https://github.com/AlexsLemonade/2020-june-training/pull/6

File: README.md (hunk 2 above)
Comment (jaclyn-taroni) [reply README.md:5]: With #6, we can probably move forward with this change.

Comment (jashapiro) [reply README.md:5]: #6 is broken, hold on.

File: README.md (hunk 1 above)
Comment (jashapiro) [reply README.md]: I wonder if it is the colon...

Comment (jaclyn-taroni) [reply README.md]: Yea that would make sense

Comment (jaclyn-taroni) [reply README.md]: https://github.com/AlexsLemonade/2020-june-training/pull/5/commits/5c1822035c09f51ef4ca32d4acfb61d22092f707 did not appear to fix it

**Pull request merged**